cli/Cargo.toml

```diff
@@ -155,6 +155,7 @@ thiserror.workspace = true
 tokio.workspace = true
 tokio-util.workspace = true
 tower-lsp.workspace = true
+tracing = { version = "0.1", features = ["log", "default"] }
```

Comment:
for reviewer: although we don't use `tracing` crate directly, we need to put this as a dependency to enable a certain feature for transitive dependencies.